testlist/other.dat

```diff
@@ -0,0 +1,28 @@
+# We do not have a toolchain for avr32 outside of Microchip login wall.
+# The work was never upstreamed to GCC.
+
+/avr
+-avr32dev1:nsh
+-avr32dev1:ostest
+
+/renesas/rx65n/rx65n-grrose
+-rx65n-grrose:ipv6
+
+/renesas/rx65n/rx65n-rsk2mb
+-rx65n-rsk2mb:ipv6
+
+# PINGUINOL toolchain doesn't provide macOS binaries
+# with the same name
+# /mips,CONFIG_MIPS32_TOOLCHAIN_PINGUINOL
+
+/risc-v,CONFIG_RV32IM_TOOLCHAIN_GNU_RVGL
+-gapuino:nsh
+-nr5m100-nexys4:nsh
+
+# x86_64-elf-gcc from homebrew doesn't seem to
+# provide __udivdi3 etc for -m32
+# /x86
```

Comment:
We should be able to re-enable this now with the Darwin exception.

Reply:
Done, I move this change to another PR.